public/static/js/task.js

```diff
@@ -400,6 +400,25 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
         return orderedTestStatuses.indexOf(task.test_result.status);
       }
 
+      // filter tests in a task based on their display name
+      $scope.filterTests = function() {
+        if($scope.task.searchField != "")
+        {
+          $scope.task.filtered_results = []
+          $scope.task.test_results.forEach(function(result){
+            var name = result.test_result.display_name;
```

Comment:
Using the "var" keyword to declare a variable will have its scope hoisted so that it's scoped to the nearest enclosing function. It doesn't cause any problems here, but to make sure that the name variable is only scoped to this if block, you should declare it with "let"

Reply:
Fixed